pkg/kubectl/cmd/cp.go

```diff
@@ -195,6 +195,11 @@ func makeTar(filepath string, writer io.Writer) error {
 	// TODO: use compression here?
 	tarWriter := tar.NewWriter(writer)
 	defer tarWriter.Close()
+
+	if strings.HasSuffix(filepath, "/") && filepath != "/" {
```

Comment:
Rather than this, just do:

```go
filepath = path.Clean(filepath)
```

or just inline that:
```go
return recursiveTar(path.Dir(path.Clean(filepath)), ...)
```

Reply:
Good.
path.Clean() will not truncate "/" into empty string.

Reply:
good! thanks